script/runc.spec

```diff
@@ -0,0 +1,63 @@
+##  Build runc RPM.
+##   1. Copy runc project tarball into ~/rpmbuild/BUILD/runc.tar.gz
+##   2. run rpmbuild -ba script/runc-centos.spec  -> will build source and binary package both.
+##
+##  Note:
+##   1. tarball must be a git project and will checkout the version while building rpms.
+##   2. this spec will build runc in dynamic link not static (if static is need, just uncomment build rules)
+##   3. need to update the TAG of runc project if release a rpm package.
+##
+%global _runc_sbindir /usr/local/sbin
+%global _runc_mandir  /usr/local/share/man
+%global _docdir  /usr/local/share/doc
```

Comment:
Are you using /usr/local to keep files installed separate from rest of the distro?

Fedora rpm convention would be to just use %{_sbindir}, %{_mandir} and so on